tests/scsi/008

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2022 Google LLC
```

Comment:
Please confirm that 2022 is the year to claim the copyright.

Reply:
Thanks for the feedback. This has been fixed.